### PR TITLE
Display fiat value in AccountList

### PR DESCRIPTION
**Pull request by FrederikBolding**
🎉 🎉 🎉

## Description

Displayed fiat value in AccountList used on the Dashboard and Settings page

## Changes

- Added `totalFiat` method to `StoreProvider`
- Displayed fiat value in AccountList using `totalFiat`

## Steps to Test

View either the Dashboard or Settings page 😄 

## Quality Assurance

- [x] The branch name is in lowercase-kebab-case with no prefix (unless it was created from Clubhouse)
- [x] The base branch is develop or gau (no nested branches)
- [x] This is related to a maximum of one Clubhouse story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)
- [x] If code is copied from existing directories, there is an explanation of why this is necesary in the description/changes, and all copying is done in separate commits to make them easy to filter out

### CODE REVIEW

File: common/v2/services/Store/StoreProvider.tsx [hunk 1]
```diff
@@ -61,6 +64,13 @@ export const StoreProvider = ({ children }: { children: React.ReactNode }) => {
       selectedAssets.filter((asset: StoreAsset) => asset.type !== 'base'),
     totals: (selectedAccounts = state.accounts) =>
       Object.values(getTotalByAsset(state.assets(selectedAccounts))),
+    totalFiat: (selectedAccounts = state.accounts) => {
+      return (getRate: (ticker: TTicker) => number | undefined) => {
+        return state.totals(selectedAccounts).reduce((sum, asset) => {
+          return (sum += convertToFiat(asset.balance, getRate(asset.ticker as TTicker)));
+        }, 0);
+      };
+    },
```
Comment (vilsbole) [inline common/v2/services/Store/StoreProvider.tsx]: How about using implicit return to reduce indentation? 
```
(selectedAccounts = state.accounts) => (
  getRate: (ticker: TTicker) => number | undefined
) =>
  state
    .totals(selectedAccounts)
    .reduce(
      (sum, asset) => (sum += convertToFiat(asset.balance, getRate(asset.ticker as TTicker))),
      0
    );
```

File: common/v2/components/AccountList.tsx [hunk 2]
```diff
@@ -133,7 +127,10 @@ function buildAccountTable(
         <Network key={index} color="#a682ff">
           {account.networkId}
         </Network>,
-        <Typography key={index}>{getBalanceFromAccount(account)}</Typography>
+        <Typography key={index}>
+          {Fiats[0].symbol}
+          {total.toFixed(2)}
+        </Typography>
```
Comment (vilsbole) [inline common/v2/components/AccountList.tsx]: 1. Use `v2/components/currency` for the display.
2. I find it troublesome to do `Fiats[0]`. How about changing the export to an object and access it as `Fiats[USER_FIAT]` (with a better name of course).

NB. It seems that we don't need to have fiats in the localstorage. We should be able to remove it from initialisation all together. I'll ask in the slack

File: common/v2/config/cacheData.ts [hunk 3]
```diff
@@ -3,26 +3,35 @@ import { Asset, Contract, NetworkId } from 'v2/types';
...
 };
 export const GBP = {
   code: 'GBP',
-  name: 'British Pounds'
+  name: 'British Pounds',
+  symbol: '£' as TSymbol
 };
 
-export const Fiats: Fiat[] = [USD, EUR, GBP];
+export const Fiats: FiatObject = { USD: USD, EUR: EUR, GBP: GBP };
```
Comment (vilsbole) [inline common/v2/config/cacheData.ts]: Travis tslint complains here: 
`expected property shorthand`

prefer: { USD, EUR, GBP }

Comment (FrederikBolding) [reply common/v2/config/cacheData.ts]: My bad, there you go! 😄

**Pull request merged**